packages/asc-ui/src/components/InfoBanner/InfoBannerStyle.ts

```diff
@@ -0,0 +1,93 @@
+import styled, { css, Theme } from '@datapunt/asc-core'
+import {
+  focusStyleOutline,
+  svgFill,
+  themeColor,
+  themeSpacing,
+} from '../../utils'
+import Button from '../Button'
+
+export type Level = 'normal' | 'attention' | 'error'
+
+type SharedProps = {
+  compact?: boolean
+}
+
+export type Props = {
+  level?: Level
+  heading?: string
+  onDismiss?: () => void
+  content?: string
+  dismissible?: boolean
+  linkProps?: {
+    href: string
+    title: string
+  }
+} & SharedProps
+
+const colorMap: Record<
+  Level,
+  ({ theme }: { theme: Theme.ThemeInterface }) => string
```

Comment:
We're repeating this a lot in different places, perhaps we should make a `WithTheme` type in the future.

Reply:
I agree!